### ISSUE TITLE
Help Wanted - Energetic / Non Hardsurface Shading

### ISSUE THREAD

**Issue by MeyerAdrian** [buddhi1980/mandelbulber2#692]
Hey guys,
not sure if you actually prefer such stuff here or in fractalforums.org?
Let me know and I will move it, if this space should be limited to pure technical issues...

I'm hoping for some advice as I'm a bit stuck at the moment...
I have a frcatal which I like very much shapewise.
But I want to achieve a very airy / non hardsurface energetic look.
This is what I have currently:
![creature_shapes_sym_v004_am](https://user-images.githubusercontent.com/45898930/61704522-2ca67a80-ad44-11e9-90cd-cf3cfec8c36e.jpg)
![creature_shapes_sym_distfog_v004_am](https://user-images.githubusercontent.com/45898930/61704531-303a0180-ad44-11e9-9dcb-5fc36bbcf1a5.jpg)

Her some reference images.
![Biolumi_pearlescent](https://user-images.githubusercontent.com/45898930/61704838-edc4f480-ad44-11e9-84be-1859c53aef2d.png)
![deep_rendered_buddhabrot_1_by_kram1032-d8w2pyj](https://user-images.githubusercontent.com/45898930/61704868-fc131080-ad44-11e9-8ab6-8d76cf285698.png)
![nebula_brot_2_by_krzysztofmarczak](https://user-images.githubusercontent.com/45898930/61704874-01705b00-ad45-11e9-90c4-ef28a97937e1.jpg)
![endofunctor_subspace_of_contingency_by_terraben-dcpikbp_s](https://user-images.githubusercontent.com/45898930/61704919-1baa3900-ad45-11e9-902b-f3ecfeee4b39.jpg)
![ct_multi_julia_3_space_p17_by_fractallife247-d8q6194](https://user-images.githubusercontent.com/45898930/61704970-311f6300-ad45-11e9-84e0-4de962be0f87.jpg)

I tried several things but could not fully get where I wanted.
Things I tried:
-Fake Lights based on Orbit Traps (seems to only work well when rendering a fractal from the outside, or at least I can't get it to work properly for this hybrid.)

- Fog based on iteration count (same here, could not really get this working with that fractal)

- Coloring the Object so that its mostly very dark (so you dont see tha hardsurface part) and some parts (kind of stripes) being very bright or even with luminosity.
Problem here is that I can't get the color palette to follow the form of the fractal (I'm using Orbit Trap z.Dot(point) but also tried others. It always looks like its a random color noise not following the shapes and directions of the fractal. With other fractals this often works very naturally and you dont even have to do a lot.
![creature_shapes_sym_color_not_following_shapes_v004_am](https://user-images.githubusercontent.com/45898930/61704549-392ad300-ad44-11e9-8552-c2079d632343.jpg)

Do you have any tips on how to achieve a look thats more airy / luminous / non hardsurface?
And maybe a tip on how I can get the colors follow mor the shape of the fractal and not be so random?

Also I was wondering if it wouldn't be a super cool feature to be able to render the fractal itself as a fog effect. So having no Hard surface shading at all. That would allow to make pretty spacey and cloudy looking things... what do you think?
![big-hero-6-disneyscreencaps com-10092](https://user-images.githubusercontent.com/45898930/61704621-68414480-ad44-11e9-931d-d98b2a3aa8e4.jpg)

Looking forward to some tips of experienced MDB users (-;
thanks a lot in advance.

.fract file
[creature_shapes_sym_v004_am.txt](https://github.com/buddhi1980/mandelbulber2/files/3421516/creature_shapes_sym_v004_am.txt)

**Comment by mclarekin** [comment]
I will answer some. Most people experienced with the artistic use of
effects are  at the Mandelbulber facebook site.

 At  fractalsforums there is currently no mandelbulber users that i  can
think of, to give you the advice you need.  There is only  one person who
seeks my advice there

A few Mandelbulber users  look at out github site, mainly to view
development progress and  to access dev versions of the program.
Some others report bugs or to document enhancements that they would like
implemented

Color is a long story, but the most common way is orbit trap standard.  Z
dot point is the hardest to use. Abox type have their own color tools that
can be blended with orbit traps or used on their own

You sometimes can  make fractals out of fog, with specific settings

In summary, Mandlbulber Facebook is the place for help

**Comment by MeyerAdrian** [comment]
Alright I'll post it there then.
Thanks.

> You sometimes can  make fractals out of fog, with specific settings
But what are you refering here to? Is it possible in Mandelbulber?

**Comment by mclarekin** [comment]
kind of, it the points are so sparse, then all you see is the fog generated from those points and not the points themselves.  It is difficult to quickly explain.  

Fog enhancements are on the near infinite TODO list

**Comment by mclarekin** [comment]
I do not know much about the artistic side of the program, but I see that you are already getting help at Facebook, so I will close this issue.
